engine/src/widget.lcb

```diff
@@ -464,6 +465,27 @@ begin
   MCWidgetEvalInEditMode(output)
 end syntax
 
+/**
+Summary:	Causes all property triggers to be triggered.
+
+Example:
+	handler TextChangedCallback()
+        UpdateTextProperty()
+        trigger all
+	end handler
+
+Description:
+Use trigger all to cause all property triggers to be triggered, for
```

Comment:
Use `trigger all` to cause the triggers on all of the widget's property to fire. For example, if a native layer which a widget is using notifies of a change in its state, and that state is available through a widget property, use `trigger all` to ensure that the IDE is notified on the change in the property.

Currently, the only property trigger which exists is the property listener mechanism in the IDE, used by such components as the property inspector to keep in sync with actual control property state.